src/client/rewards/Reports/Reports.js

```diff
@@ -9,16 +9,24 @@ import './Reports.less';
 
 const Reports = ({ intl, userName }) => {
   const [sponsors, setSponsors] = useState({});
+  const [isHive, setIsHive] = useState(true);
 
   const requestParams = {
     sponsor: userName,
     globalReport: true,
   };
 
-  const getHistories = params => {
+  const getIsHive = currency => {
+    if (currency === 'hive') return true;
+    return false;
+  };
```

Comment:
you can write function in this way:
`const getIsHive = currency => currency === 'hive';`